### PR TITLE
[4/11] Page Header for components health

### PR DESCRIPTION
**Pull request by rcorral**
---
~~⚠️   _This branch depends on features that will be introduced with #1482_~~ Merged

---

### CODE REVIEW

**Comment by jfurrow** [comment]
I pushed a commit with a few modifications; otherwise it LGTM!

File: src/js/pages/system/UnitsHealthDetail.js [hunk 1]
```diff
@@ -138,49 +146,52 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
...
+    let content = null;
+    let unit = null;
 
     if (hasError) {
-      return this.getErrorNotice();
-    }
-
-    if (isLoadingUnit || isLoadingNodes) {
-      return this.getLoadingScreen();
+      content = this.getErrorNotice();
```
Comment (mlunoe) [inline src/js/pages/system/UnitsHealthDetail.js]: Could we turn this into a separate function (e.g. `this.getContent()`) so we can `return` and keep them as simple `if`-statements?

File: src/js/pages/system/UnitsHealthNodeDetail.js [hunk 2]
```diff
@@ -68,36 +100,40 @@ class UnitsHealthNodeDetail extends mixin(StoreMixin) {
 
   render() {
     let {hasError, isLoadingNode, isLoadingUnit} = this.state;
+    let content = null;
+    let node = null;
+    let unit = null;
 
     if (hasError) {
-      return this.getErrorNotice();
+      content = this.getErrorNotice();
```
Comment (mlunoe) [inline src/js/pages/system/UnitsHealthNodeDetail.js]: Same here, let's create a separate function to get this

**Comment by MatApple** [comment]
@mesosphere-ci retest this please

**Pull request merged**